bin/fuel-core/src/cli/run.rs

```diff
@@ -458,6 +452,8 @@ impl Command {
             database_type,
             max_database_cache_size,
             #[cfg(feature = "rocksdb")]
+            columns_policy: Default::default(),
```

Comment:
It would be nice to control policy explicitly in the `fuel-core-bin` in the case of production, or non production

Reply:
Done

Reply:
I meant to set it here `OnCreation` if "production" feature is used or `Lazy`, if it is not used

Reply:
Done.